### PR TITLE
Update devquests.md

### PR DESCRIPTION
**Pull request by CarnavalSines**
Added new DQ22 ruling, updated DQ2 using the wrong badge

### CODE REVIEW

File: docs/developer-docs/devquests.md [hunk 1]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
 
+### 22 [This Belonged in a Museum!](https://retroachievements.org/game/25678)
+
+![DQ022](https://retroachievements.org/Images/102134.png)
+
+**Type**: Revision, Maintenance 
+**Goal**: Obtain 4 points by dusting off older sets, bringing them up to current RA standards  
+**Repeatable**: No
+
+1. The Quest is completed upon obtaining 4 points towards it. Full Set Dust-Off = 1 Point, Misc Task = 0,25pt, at least one set dust-off required for the badge.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]: Probably should spell it out as 0.25 points (American-centric, I know), or as "a quarter point." Or use the 1/4 Unicode character.

Comment (CarnavalSines) [reply docs/developer-docs/devquests.md]: Per internal DQ Team talk, updating the point framework to require 12 Total, meaning the dust-off is now worth 4pts and the misc 1pt each.

File: docs/developer-docs/devquests.md [hunk 2]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
 
+### 22 [This Belonged in a Museum!](https://retroachievements.org/game/25678)
+
+![DQ022](https://retroachievements.org/Images/102134.png)
+
+**Type**: Revision, Maintenance 
+**Goal**: Obtain 12 Points by dusting off older sets, bringing them up to current RA standards  
+**Repeatable**: No
+
+1. The Quest is completed upon obtaining 12 points towards it. Full Set Dust-Off = 4 Points, Misc Task = 1 Point, at least one set dust-off required for the badge.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]: Consider making the point descriptions sub-bullets for more visibility.

File: docs/developer-docs/devquests.md [hunk 3]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
+3. Misc Tasks: Update a set in order to remove it from the [No Dynamic RP](https://retroachievements.org/game/3083), [Player-Input RP](https://retroachievements.org/game/28506) or [Single Save Support](https://retroachievements.org/game/28449) Hubs. Doesn't stack with a Full Set Clearing.
+4. No Collaboration allowed.
+5. This Quest do not stack with other revision DevQuest. If a set would also be eligible to DQ2 or DQ13 they will take priority.
+6. Sets are subject to DevQuest team approval, along with a small plan on what the dust-off will be about. Sets already in the approved DQ22 list do not require approval, only notification.
+7. Badges update must go through the Icon-Gauntlet revision process.
+8. Revision rules apply as usual, meaning if you only clean-up the logic and don't change the achievements scope you don't need one.
+9. Devs own sets made prior to 2024 are eligibles for half the points on a Full Dust-Off. Similarly to sets not pre-approved, they will need to be sent to the DevQuest team for approval.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]:
```suggestion
9. Devs' own sets made prior to 2024 are eligible for half the points on a Full Dust-Off. As with sets that are not pre-approved, they will need to be sent to the DevQuest team for approval.
```

File: docs/developer-docs/devquests.md [hunk 4]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
+### 22 [This Belonged in a Museum!](https://retroachievements.org/game/25678)
+
+![DQ022](https://retroachievements.org/Images/102134.png)
+
+**Type**: Revision, Maintenance 
+**Goal**: Obtain 12 Points by dusting off older sets, bringing them up to current RA standards  
+**Repeatable**: No
+
+1. The Quest is completed upon obtaining 12 points towards it. Full Set Dust-Off = 4 Points, Misc Task = 1 Point, at least one set dust-off required for the badge.
+2. Full Set Dust-Off: Clean up old code notes, update achievement logics for modern best practices (removing unnecessary Hits and Resets to use Delta checks instead for example), new cleaner badges, updated descriptions and achievements titles, new dynamic RP.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]:
```suggestion
2. Full Set Dust-Off: Clean up old code notes, update achievement logic for modern best practices (removing unnecessary Hits and Resets to use Delta checks instead, for example), update with cleaner badges, update descriptions and achievement titles, new dynamic RP.
```
Just some subject/verb tense agreement updates

File: docs/developer-docs/devquests.md [hunk 5]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
+3. Misc Tasks: Update a set in order to remove it from the [No Dynamic RP](https://retroachievements.org/game/3083), [Player-Input RP](https://retroachievements.org/game/28506) or [Single Save Support](https://retroachievements.org/game/28449) Hubs. Doesn't stack with a Full Set Clearing.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]: Consider putting the three hubs as sub-bullets for readability

File: docs/developer-docs/devquests.md [hunk 6]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
+4. No Collaboration allowed.
+5. This Quest do not stack with other revision DevQuest. If a set would also be eligible to DQ2 or DQ13 they will take priority.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]:
```suggestion
5. This quest does not stack with other revision DevQuests. If a set would also be eligible to DQ2 or DQ13 those quests will take priority.
```

File: docs/developer-docs/devquests.md [hunk 7]
```diff
@@ -408,6 +408,25 @@ You must have made sets for at least 15 subgenres before opt-in. To opt-in, make
...
+6. Sets are subject to DevQuest team approval, along with a small plan on what the dust-off will be about. Sets already in the approved DQ22 list do not require approval, only notification.
+7. Badges update must go through the Icon-Gauntlet revision process.
```
Comment (redwizard42) [inline docs/developer-docs/devquests.md]:
```suggestion
7. Badge updates must go through the Icon-Gauntlet revision process.
```

File: docs/developer-docs/devquests.md (hunk 2 above)
Comment (redwizard42) [inline docs/developer-docs/devquests.md]:
```suggestion
1. The quest is completed upon obtaining 12 points towards its goals. Full Set Dust-Off = 4 Points, Miscellaneous Task = 1 Point, at least one set dust-off required for the badge.
```

Comment (CarnavalSines) [reply docs/developer-docs/devquests.md]: Done

File: docs/developer-docs/devquests.md (hunk 5 above)
Comment (CarnavalSines) [reply docs/developer-docs/devquests.md]: Done

**Review by redwizard42 — APPROVED** [review]

**Pull request merged**